causing/utils.py

```diff
@@ -348,10 +348,9 @@ def vecmat(mz):
 
 def render_dot(dot_str, filename):
     """render Graphviz graph from dot_str to svg using pydot"""
```

Comment:
The docstring says `pydot`, which is not accurate anymore (we don't have to mention the library IMO).